scripts/instance/mask_rcnn/train_mask_rcnn.py

```diff
@@ -34,16 +35,7 @@
 from multiprocessing import Process
 from gluoncv.model_zoo.rcnn.mask_rcnn.data_parallel import ForwardBackwardTask
 
-try:
-    import horovod.mxnet as hvd
-except ImportError:
-    hvd = None
-
-try:
-    from mpi4py import MPI
-except ImportError:
-    logging.info('mpi4py is not installed. Use "pip install --no-cache mpi4py" to install')
-    MPI = None
+import herring.mxnet as hvd
```

Comment:
why are we still using horovod name everywhere?

Reply:
The agenda is to minimize the existing code as less as possible to make it compatible with herring